include/cell.tcc

```diff
@@ -844,3 +844,107 @@ template <unsigned Tdim>
 inline unsigned mpm::Cell<Tdim>::previous_mpirank() const {
   return this->previous_mpirank_;
 }
+
+//! Map cell volume to nodes
+template <unsigned Tdim>
+bool mpm::Cell<Tdim>::map_cell_volume_to_nodes(unsigned phase) {
+  bool status = true;
+  try {
```

Comment:
We don't need to use exception handling here, instead, use assert. A failure here will give incorrect results, so this should fail.